src/Traversal/Sdk/Sdk.targets

```diff
@@ -18,6 +18,15 @@
     <OutputPath Condition=" '$(Configuration)' != '' And '$(Platform)' != '' ">bin\$(Configuration)\$(Platform)\</OutputPath>
   </PropertyGroup>
 
+  <!--
+    TargetFrameworks is used to convey that a project targets multiple frameworks.  Traversal does not support multi-targeting
```

Comment:
>  Traversal does not support multi-targeting

@jeffkl what does this mean? Can the `ProjectReference` projects can still use `TargetFrameworks` with multiple values?

Reply:
Traversal projects don't really support a target framework, since they aren't built into an assembly that is bound to a particular framework.  This means that things like NuGet do not need to confirm that the target framework is compatible, the traversal project is just basically telling the system to build a set of projects.  The ProjectReference in an actual project does need to run through a compatibility check since it will be a loadable assembly though.

But ProjectReference items in traversal aren't the same, we just use the same item for consistency.  But yes, actual projects still get to use multiple target frameworks.